packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -187,6 +196,16 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
     });
   }
 
+  KeyEventResult handleKeyEvent(FocusNode focusNode, RawKeyEvent event) {
+    final game = widget.game;
+
+    if (game is KeyboardEvents) {
+      return game.onKeyEvent(event, RawKeyboard.instance.keysPressed);
+    }
+
+    return KeyEventResult.skipRemainingHandlers;
```

Comment:
Just wondering if here shouldn't be `KeyEventResult.handled`?

I was testing this Focus widget on another project, and was using `skipRemainingHandlers`, I was getting that bell noise when I kept a key pressed. Changing to `handled` the noise stoped.